### PR TITLE
Add example JSON snippets to transports

### PR DESCRIPTION
**Pull request by bachp**
@quartzjer I didn't know what would go into the signal for WebRTC. If you can tell me I will replace the TODO. If not I will add it later.

### CODE REVIEW

File: v3/transports/http.md [hunk 1]
```diff
@@ -13,4 +13,13 @@ By default all packets should be sent as `volatile` events, but handshakes may b
 
 A new keepalive handshake may be automatically triggered when no packets have been sent for 60 seconds as an extra validation that the socket.io connection is valid.
 
+## JSON
```
Comment (quartzjer) [inline v3/transports/http.md]: Would the section title be better as "Path JSON"?

File: v3/transports/http.md [hunk 2]
```diff
@@ -13,4 +13,13 @@ By default all packets should be sent as `volatile` events, but handshakes may b
...
 
+Example JSON
+
+```json
+{
+    "url": "https://http://192.168.0.36:42424",
```
Comment (quartzjer) [inline v3/transports/http.md]: double http :)

File: v3/transports/webrtc.md [hunk 3]
```diff
@@ -9,4 +9,13 @@ When an endpoint supports this it includes in it's `paths` a `{"type":"webrtc"}`
 
 All packets must be [cloaked](../e3x/cloaking.md) and are sent as base64 encoded message events over a data channel.
 
+## JSON
 
+Example JSON:
+
+```json
+{
+    "signal": "TODO:What goes here",
```
Comment (quartzjer) [inline v3/transports/webrtc.md]: Due to the nature of WebRTC and signals being dynamic, the path json is nothing other than a `{"type":"webrtc"}` to signal support of the transport.  There may be some helpful static hints that could be included but I haven't encountered them yet, as everything is negotiated via the signaling during the actual peer connection setup :)

**Comment by quartzjer** [comment]
If any of those make sense just update or reply and I'll get this merged, good stuff :+1:

**Comment by bachp** [comment]
I updated the commit according your comment. Should be good to merge now.
Just one thing: I was wondering if there are path json examples for other transports as-well (tls, bluetooth,802.15.4?), because there I didn't find any examples.

File: v3/transports/webrtc.md [hunk 4]
```diff
@@ -9,4 +9,12 @@ When an endpoint supports this it includes in it's `paths` a `{"type":"webrtc"}`
 
 All packets must be [cloaked](../e3x/cloaking.md) and are sent as base64 encoded message events over a data channel.
 
+## Path JSON
 
+Example [path](../channels/path.md) JSON:
+
+```json
+{
+    "signal": "TODO:What goes here",
```
Comment (quartzjer) [inline v3/transports/webrtc.md]: oop, removed the wrong line? is just `{"type":"webrtc"}` and no signal field :)

**Comment by quartzjer** [comment]
Right, those are just stubs, suggestions on a good way to signal that those transports are planned in the docs?

There's issues for most things that are stubs already, but it's nice to have some pointer to that state in the spec in the right places as well.

File: v3/transports/webrtc.md (hunk 4 above)
Comment (bachp) [inline v3/transports/webrtc.md]: yes I did ;) Will fix this tonight.

**Comment by bachp** [comment]
I fixed my wrong fix ;)
Just to make sure I understand your last commit correctly.
By "pointers in the spec" you mean TODOs?

I have quite good experience with TODO: as it allows to easily grep for it in the source.

**Comment by quartzjer** [comment]
Yeah, in a protocol spec it's more of a "draft" state than a code TODO though, I'm going to file an issue to do a consistency check on all the draft aspects and flag them.

**Pull request merged**